modules/beta-autopilot-public-cluster/cluster.tf

```diff
@@ -39,6 +39,9 @@ resource "google_container_cluster" "primary" {
       channel = release_channel.value.channel
     }
   }
+  cost_management_config {
```

Comment:
Is this supported for AP clusters?

Reply:
Yes it is supported by AP Clusters as well